source/features/create-release-shortcut.tsx

```diff
@@ -22,4 +27,10 @@ void features.add({
 		pageDetect.isReleasesOrTags
 	],
 	init
+}, {
+	include: [
+		pageDetect.isReleasesOrTags,
```

Comment:
I guess this is the only change left
```suggestion
		pageDetect.isEditingRelease,
```

Reply:
>Note pageDetect.isReleasesOrTags is needed since if there is an error publishing the release it does not stay on the new page

Try by going to the release page dont fill in any info and click publish

Reply:
And

Reply:
JS comment please